examples/knobs_example/pubspec.lock

```diff
@@ -223,14 +223,14 @@ packages:
       path: "../../packages/widgetbook"
       relative: true
     source: path
-    version: "3.0.0-beta.3"
+    version: "3.0.0-beta.4"
```

Comment:
version `3.0.0-beta.4` has a navigation problem for the properties. https://github.com/widgetbook/widgetbook/pull/418 fixes that but I already deployed `3.0.0-beta.5` so you might want to run `melos bs` again

Reply:
I ran `melos bs` and pushed again but I'm not sure if it's right to include `pubspec` changes of other packages in this PR. Let me know if that's an issue and I can revert that.